llvm-general/src/LLVM/General/Internal/OrcJIT/IRCompileLayer.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+module LLVM.General.Internal.OrcJIT.IRCompileLayer where
+
+import           LLVM.General.Prelude
```

Comment:
Fixed, sorry for that. I need to figure out how to stop my emacs shortcut for sorting imports from aligning them.